swagger/v1/api_spec.json

```diff
@@ -1718,7 +1718,7 @@
           },
           "per_page": {
             "type": "integer",
-            "description": "The number items to display on a page. Defaults to 100. Maximum is 250, if the value is greater that the maximum allowed it will fallback to 250.",
+            "description": "The number items to display on a page. Defaults to 250. Maximum is 3000, if the value is greater that the maximum allowed it will fallback to 250.",
```

Comment:
the `default_per_page` is still set to 100, this bit also needs changing:
```
if the value is greater that the maximum allowed it will fallback to ~250~ 3000
```

Reply:
Good spot. I knew there wasn't something right about that.